code-studio/src/js/initializeCodeMirror.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file Function that initializes a CodeMirror editor in place of a textarea.
+ */
+/* global inlineAttach */
+'use strict';
```

Comment:
Was use strict added for a specific reason? Should we be using this more places. For some reason, I thought that got added automatically to modules by browserify, but I might be making that up completely.